### DISCUSSION TITLE
Changing the position of React Flow Attribution

### DISCUSSION THREAD

**Discussion by sodatta119** [xyflow/xyflow#4812] [Q&A]
We are using React Flow at our organization and want to change the position of "React Flow" attribution, by placing it in the footer as "Powered By React Flow". I want to know if this is permitted by ReactFlow?

**Comment by chrtze** [comment]
Yes, that's fine. The library is MIT-licensed so you can remove the attribution or put it somewhere else. You can find more information here: https://reactflow.dev/learn/troubleshooting/remove-attribution

**Reply by sodatta119** [reply]
Thank you!
